src/Tasks/ProductImportTask.php

```diff
@@ -46,14 +47,21 @@ public function getSchedule()
 
     /**
      * Import handler for cron task
+     * @throws FlowException
      */
     public function process()
     {
-        $this->run(new NullHTTPRequest());
+        try {
```

Comment:
Instead of creating a NullHTTPRequest which does nothing, flip the order (run() calls process() instead of the other way around).

Reply:
You should probably move the FlowException handling into the same path as the code itself.